src/Makefile.libavcenclib

```diff
@@ -0,0 +1,70 @@
+# Copyright 2014 Google.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Compiles the libavc encoder library.
+# Assumes that the current directory is "libavc".
+# TODO(hta): Figure out how to merge the two Makefiles together.
+#
+my-dir = ../libavc
+
+SOURCE=../libavc
+ARCH=x86
+include $(SOURCE)/encoder.mk
+include $(SOURCE)/encoder.x86_64.mk
+
+
+CFLAGS+= -O3 -DNDEBUG -D_MBCS
+CFLAGS +=-fPIC

```

Comment:
Nit: consistency with spaces before/after = and += operators? Shell would care; I guess make doesn't, or this would not work, would it?